### PR TITLE
Extending star rating component color

### PR DESCRIPTION
**Pull request by sirLisko**
I want to be able to specify the colour of the *Stars* in the `StarRating` component.

It also adds `prop-types` check instead on _flow_.

Screenshot 2019-05-09 at 14 29 05

### How to test it
1. check out the branch
1. `npm install`
1. spin up the style guide
1. check that the `star rating` component looks ok

### CODE REVIEW

File: src/components/star-rating/star-rating.react.js [hunk 1]
```diff
@@ -8,14 +8,8 @@ import {UIBase} from '../layout';
...
-type TProps = {
-  score: Number,
-  component?: string,
-  className?: string | Object,
-};
-
-export default function StarRating(props: TProps) {
-  const {score = 0, className, ...otherProps} = props;
+function StarRating(props) {
+  const {score = 0, className, color, ...otherProps} = props;
```
Comment (leesus) [inline src/components/star-rating/star-rating.react.js]: This should have a default for the `color` prop.

Comment (sirLisko) [reply src/components/star-rating/star-rating.react.js]: Good idea, at the moment the colour is defined in the css, but actually makes more sense having it centralised in the component 👍

Comment (sirLisko) [reply src/components/star-rating/star-rating.react.js]: Done ;)

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kalo/ui",
-  "version": "3.8.1",
+  "version": "3.9.0",
```
Comment (hereticsun) [inline package.json]: No need to make this change manually. It is done as part of the release process

Comment (sirLisko) [reply package.json]: Thanks for the heads up :)
I got rid of that commit 👍

**Comment by mroderick** [comment]
I assume this component doesn't have a vanilla equivalent, which is why it's OK to remove the default colour from the CSS?

**Comment by sirLisko** [comment]
According to the documentation and the codebase it doesn't.

Happy to think about an alternative solution if needed.

**Comment by mroderick** [comment]
> According to the documentation and the codebase it doesn't.
> 
> Happy to think about an alternative solution if needed.

Thanks you for looking that up 👍 
Please go ahead with this.

**Comment by mroderick** [comment]
Oh, it was already merged :)

**Pull request merged**